src/Input.cpp

```diff
@@ -140,39 +140,42 @@ struct Gosu::Input::Impl : private Gosu::Noncopyable
                 }
                 int gamepad_slot = -1;
                 int joystick_instance_id = -1;
+                int i = e->jdevice.which; // SDL2 device_index
 
-                // Loop through attached gamepads as e->jdevice.which cannot be trusted (always 0)
```

Comment:
This comment was introduced here: https://github.com/gosu/gosu/pull/543

Soo...does this mean we can actually trust e->jdevice.which now? Was it only broken in the 18.04 version?